packages/plugins/build-report/src/helpers.ts

```diff
@@ -22,6 +22,7 @@ import path from 'path';
 const EXTENSION_RX = /\.(?!.*(?:\.|\/|\\))(\w{1,})/g;
 
 // Will match any type of query characters.
+// ? or %3F or |.
```

Comment:
what is `%3F`?

Also you should remove the `.` from this comment as I first understood that as "will match `|.`"

Reply:
`%3F` is `?` url encoded.